src/test/java/org/broadinstitute/hellbender/utils/hmm/HeavyStateTestHMModel.java

```diff
@@ -18,13 +21,9 @@
     private final double lightStateProbs;
 
     public HeavyStateTestHMModel(final int numStates, final int heavyState, final double heavyStateWeight) {
-        if (numStates <= 1) {
-            throw new IllegalArgumentException("bad number of states");
-        } else if (heavyState < 0 || heavyState >= numStates) {
-            throw new IllegalArgumentException("bad heavy state");
-        } else if (heavyStateWeight <= (1.0 / numStates) || heavyStateWeight >= 1.0) {
-            throw new IllegalArgumentException("bad heavy state weight");
-        }
+        Utils.validateArg(numStates > 1, "bad number of states");
+        Utils.validIndex(heavyState, numStates);
+        ParamUtils.inRange(heavyStateWeight, (1.0 / numStates), 1.0, "bad heavy state weight");
```

Comment:
`ParamUtils.inRange` accepts a closed interval, whereas the original code accepted an open interval. Don't know what `heavyStateWeight` represents, but if it's equal to 1.0, I think the behavior changes; old code throws an exception, new code doesn't. 

That said, `heavyStateWeight` is a double so == comparison is not exact so maybe that doesn't actually make a difference. Either way I'd like to hear your thoughts on this